pkgs/by-name/br/breseq/package.nix

```diff
@@ -0,0 +1,82 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  callPackage,
+  nix-update-script,
+  libz,
+  autoconf,
+  automake,
+  libtool,
+  perl,
+  R,
+  bowtie2,
+  which,
+  ghostscript,
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Lets use `finalAttrs` pattern here, as it improve override flexibility:
- #315337